finalPrep/modZeroHero.js

```diff
@@ -1,52 +1,79 @@
 // Challenge - See if you can follow the instructions and complete the exercise in under 30 minutes!
 
 // Declare two variables - heroName AND specialAbility - set to strings
+var heroName = "Captain America";
+var specialAbility = "super strength";
 
 // Declare two variables - greeting AND catchphrase
 //   greeting should be assigned to a string that uses concatenation to include the heroName
 //   catchphrase should be assigned to a string that uses interpolation to include the specialAbility
+var greeting = "Hello " + heroName;
+var catchphrase = `Fear my ${specialAbility}!`;
 
 // Declare two variables - power AND energy - set to integers
+var power = 50;
+var energy = 75;
 
 // Declare two variables - fullPower AND fullEnergy
 //   fullPower should multiply your current power by 500
 //   fullEnergy should add 150 to your current energy
+var fullPower = power * 500;
+var fullEnergy = energy + 50;
 
 // Declare two variables - isHuman and identityConcealed - assigned to booleans
-
+var isHuman = true;
+var identityConcealed = false;
 
 // Declare two variables - archEnemies AND sidekicks
 //   archEnemies should be an array of at least 3 different enemy strings
 //   sidekicks should be an array of at least 3 different sidekick strings
+var archEnemies = ["Adolf Hitler", "Red Skull", "Baron Zemo"];
+var sidekicks = ["Bucky", "Falcon", "Demolition Man"];
 
 // Print the first sidekick to your console
+console.log(sidekicks[0]);
 
 // Print the last archEnemy to the console
+console.log(archEnemies[2]);
 
 // Write some code to add a new archEnemy to the archEnemies array
+archEnemies.push("Crossbones");
 
 // Print the archEnemies array to console to ensure you added a new archEnemy
+console.log(archEnemies);
 
 // Remove the first sidekick from the sidekicks array
+sidekicks.shift();
 
 // Print the sidekicks array to console to ensure you added a new sidekick
+console.log(sidekicks);
 
 // Create a function called assessSituation that takes three arguments - dangerLevel, saveTheDay, badExcuse
 //   - dangerLevel should be an integer
-//   - saveTheDay should be a string a hero would say once they save the day 
+//   - saveTheDay should be a string a hero would say once they save the day
 //   - badExcuse should be a string a hero would say if they are too afraid of the dangerLevel
 
+
 // Your function should include an if/else statement that meets the following criteria
 //   - Danger levels that are above 50 are too scary for your hero. Any danger level that is above 50 should result in printing the badExcuse to the console
 //   - Anything dangerLevel that is between 10 and 50 should result in printing the saveTheDay string to the console
 //   - If the dangerLevel is below 10, it means it is not worth your time and should result in printing the string "Meh. Hard pass." to the console.
+function assessSituation(dangerLevel, saveTheDay, badExcuse) {
+  if(dangerLevel > 50) {
+    console.log(badExcuse);
+  } else if(dangerLevel > 10) {
```

Comment:
It is best practice to avoid conditions that overlap. For example, here, a number that is over 50 is also over 10. You can use an && operator to create 2 conditions, or a trick is to save the condition with the most complex logic for your `else` block.